frameworkStructureTask/src/main/java/utils/PropertyReader.java

```diff
@@ -0,0 +1,38 @@
+package utils;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertyReader {
+
+    private static final Properties properties;
+    static {
+        properties = new Properties();
+        try{
+            properties.load(PropertyReader.class.getClassLoader().getResourceAsStream("config.properties"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+
+    public static String getHerokuAppUrl() {
+        return properties.getProperty("herokuAppUrl");
+    }
+
+    public static String getChromeBrowser() {
+        return properties.getProperty("chrome.browser");
+    }
+
+    public static String getOperaBrowser() {
+        return properties.getProperty("opera.browser");
+    }
+
+    public static String getFirefoxBrowser() {
+        return properties.getProperty("firefox.browser");
+    }
```

Comment:
Instead of these 3 methods you can create the only one like getBrowser() and take the name of the browser. In config.properties you can specity the only one property related to browser instarnce called "browser"